atak-civ/plugin-examples/plugintemplate/app/src/main/java/com/atakmap/android/plugintemplate/PluginTemplateDropDownReceiver.java

```diff
@@ -62,15 +69,25 @@ public void onClick(View v) {
                     AtakBroadcast.getInstance().sendBroadcast(i);
                 }
             }
-        });
+        });*/
+
+        //sets up the list adapter and layout manager for multiple timers
+        mainScreenTimerList = templateView.findViewById(R.id.timer_recycler);
+        LinearLayoutManager manager = new LinearLayoutManager(context);
+        mainScreenTimerList.setLayoutManager(manager);
+
+        adapter = new TimerListAdapter(timers);
+        mainScreenTimerList.setAdapter(adapter);
+
```

Comment:
I put this in the onrecieve so that it would recreate the contents of the recyclerview every time new data was received. Being in the constructor would create an empty view

Reply:
I'm really confused why I wrote this comment originally. It looks like this code is already in the constructor.